beacon-chain/entrypoint.sh

```diff
@@ -3,6 +3,37 @@
 # Concatenate EXTRA_OPTS string
 [[ -n "$CHECKPOINT_SYNC_URL" ]] && EXTRA_OPTS="${EXTRA_OPTS} --checkpoint-sync-url=${CHECKPOINT_SYNC_URL}"
 
+case $_DAPPNODE_GLOBAL_EXECUTION_CLIENT_MAINNET in
+"geth.dnp.dappnode.eth")
+    HTTP_ENGINE="http://geth.dappnode:8551"
+    ;;
+"nethermind.public.dappnode.eth")
+    HTTP_ENGINE="http://nethermind.dappnode:8551"
```

Comment:
```suggestion
    HTTP_ENGINE="http://nethermind.public.dappnode:8551"
```
Adopt for mainnet package.